src/App.jsx

```diff
@@ -27,6 +27,25 @@ function App() {
         </Route>
       </Routes>
       <>
+        <section>
+          <h1 className="home-header">Bruin Dictionary</h1>
+          <p className="home-description">
+            Welcome to Bruin Dictionary! A crowdsourced online dictionary for UCLA’s acronyms, slang
+            words, and inside jokes.
+          </p>
+          <a href="#" className="home-button">
+            Continue
+            <svg
```

Comment:
Also, i installed icon library(https://ant.design/components/icon) You can simply import any icons you want, not using svg tag. But again, don't work on this on this pr, work on another branch and push a separate pr regarding those changes